fbw-a380x/src/systems/systems-host/systems/LegacyGpws.ts

```diff
@@ -0,0 +1,814 @@
+// Note the master copy of these flags is contained in `fbw-a32nx\src\systems\shared\src\AutoCallOuts.ts`
+// Please do not edit here unless copying from there.
```

Comment:
You have disconnected this comment from the flags... it described where you could have imported them from :D.